src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -11,46 +11,30 @@ import selectionShape from '../../../propTypeShapes/selection';
 import favouritesShape from '../../../propTypeShapes/favourites';
 import windowStateShape from '../../../propTypeShapes/windowState';
 
-/*
- *  dataTransfer.getData is only available in dragstart, drop and dragEnd
```

Comment:
I think I'd keep this comment with the function, as I think it makes it easier to understand what's happening.